src/pineko/__init__.py

```diff
@@ -95,17 +91,16 @@ def load_pineappl_dy():
 operators = load_eko(operators_card)
 
 operator_grid = np.array([op["operators"] for op in operators["Q2grid"].values()])
+alpha_s = (
+    lambda q2: eko.strong_coupling.StrongCoupling.from_dict(theory_card).a_s(q2)
+    * 4
+    * np.pi
+)
+
 # for the time being replace with a fake one, for debugging
 #  operator_grid = eko_identity(operator_grid.shape)
 
-pineappl_grid_q0 = pineappl_grid.convolute_eko(
-    operators["q2_ref"],
-    alpha_s(q2_grid),
-    operators["pids"],
-    x_grid,
```

Comment:
This was an error, right? because it was coming from pineappl and so not in the order of eko, as was assumed - using https://github.com/N3PDF/pineappl/pull/51/files/114fbad8ca6291aadaf96c59065b8ac6ac45f301..c1a242435f64409d124227ff2b068a9a18f7d06e#diff-c0ce134e4c3635c8c19d90ea8ecbb1019377aeedd8c58d4cdde01b17a0eeaf66R47 we should be able to revert the fix we applied yesterday evening @AleCandido do you agree?

Reply:
The comment refers only to line 105 = xgrid

Reply:
Yes, indeed it was an error I fixed, but it's consistent with the current version of pineappl.

Reply:
https://github.com/N3PDF/pineappl/blob/c1a242435f64409d124227ff2b068a9a18f7d06e/pineappl/src/grid.rs#L1039-L1045

Reply:
https://github.com/N3PDF/pineappl/blob/c1a242435f64409d124227ff2b068a9a18f7d06e/pineappl/src/grid.rs#L1039-L1045